slices/secureboot-db.yaml

```diff
@@ -0,0 +1,19 @@
+package: secureboot-db
+
+essential:
+  - secureboot-db_copyright
+
+slices:
+  dist:
```

Comment:
wouldn't `data` be a more appropriate name? (since we already have other slices using a similar convention)